Lib/configparser.py

```diff
@@ -612,6 +612,8 @@ def __init__(self, defaults=None, dict_type=_default_dict,
             self._interpolation = self._DEFAULT_INTERPOLATION
         if self._interpolation is None:
             self._interpolation = Interpolation()
+        if not isinstance(self._interpolation, Interpolation):
+            raise TypeError("interpolation is not an instance of Interpolation")
```

Comment:
Since this is a debugging message, let's make it a little more verbose as to what is and isn't allowed. Plus, let's make it fit into 79 characters per PEP 8.

```suggestion
            raise TypeError(
                f"interpolation= must be None or an instance of Interpolation;"
                f" got an object of type {type(self._interpolation)}"
            )
```